documentation/source/user_section/tutorials/1/1-3.rst

```diff
@@ -0,0 +1,58 @@
+1.3 Hands-on Example: T2
+########################
+
+Run the following command to process the image:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_qc.png
+  :align: right
+  :figwidth: 40%
+
+  The QC report for the segmented image
+
+.. code:: sh
+
+   sct_propseg -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
```

Comment:
Love the detailed description of what all the flags mean, very clear and easy to use 😄